examples/datadog/agent_on_openshift_values.yaml

```diff
@@ -0,0 +1,44 @@
+# Datadog Agent with Logs, APM, Processes, and System Probe enabled
+# with specific configurations to work on OpenShift 4.
+# When installing the chart, install onto a non-default namespace with
+# `helm install --namespace <your_ns>` due to existing SecurityContextConstraints on the default namespace.
+# To setup the SCC in the agent namespace, update "system:serviceaccount:default:datadog-agent"
+# with "system:serviceaccount:<your_ns>:<your_sa>" in
+# https://github.com/DataDog/datadog-agent/blob/master/Dockerfiles/manifests/openshift/scc.yaml and apply.
+
+targetSystem: "linux"
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  clusterName: <CLUSTER_NAME>
+  tags: []
+  criSocketPath: /var/run/crio/crio.sock
```

Comment:
Clarify it's for OpenShift 4

Reply:
it's written in the top comments, do you think it's worth making this example for openshift 3 or 4 and including additional comments for 4 specifically?